.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 To make our job easier, please spend time to review your application before submitting. 
 
+NOTE: Your not *required* to fill in this application. 
```

Comment:
Fix the grammar, and make it a comment!
```suggestion
<!-- NOTE: You're *not* required to fill in this application. -->
```

Reply:
I don't think comment is a good idea as no one sees it.